src/EditorFeatures/CSharpTest/ExtractClass/ExtractClassTests.cs

```diff
@@ -2521,6 +2521,70 @@ class C
             }.RunAsync();
         }
 
+        [Fact]
+        [WorkItem(55610, "https://github.com/dotnet/roslyn/issues/55610")]
+        public async Task TestMultipleMethodsSelected_WithTypeContainingBaseClass()
```

Comment:
Looks like the issue was with `TryGetClassActionAsync`, so maybe have another test that select `Derived` as well?